nifi-nar-bundles/nifi-hive-bundle/nifi-hive-processors/src/main/java/org/apache/nifi/util/hive/HiveConfigurator.java

```diff
@@ -74,7 +74,8 @@ public HiveConf getConfigurationFromFiles(final String configFiles) {
 
     public void preload(Configuration configuration) {
         try {
-            FileSystem.get(configuration);
+            FileSystem.get(configuration).close();
```

Comment:
Hey @mattyb149 

The two clusters I was testing against (secure and unsecure) were both configured for HA so I did test preloading with HA inadvertently :smile: